wordpress-mobile/WordPress-Aztec-iOS
# Deletion of list markers

Ref #34 

Make sure than when deleting list markers they are delete as an all and not partially.

How to test:

- Create a list ordered/unordered
- Place the cursor just after a marker ( beginning of paragraph)
- Tap on delete/backward 

Check if the markers is deleted has all.

This PR still not upgrades the numbering of the follow up list paragraphs.
